docs/credit.html

```diff
@@ -68,6 +68,12 @@ <h4>
                               <p>
                                   The music for Wild World was collected from the playlist by P4lf located <a target="_blank" href="https://www.youtube.com/playlist?list=PLv2IGBk7nw2NaCeY04oTpcBbg9hno5Y3M">here</a>.
                               </p>
+                              <h4>
+                                  Special Thanks to: maximilian1001
+                              </h4>
+                              <p>
+                                  A huge shoutout to <a target="_blank" href="https://github.com/maximilian1001">maximilian1001</a>, who made it his life mission to relentlessly remind me for months until I finally fixed the <a target="_blank" href="https://github.com/lhns/newleafradio/issues/4">loading issues</a>. Without his unyielding pressure, his charmingly persistent messages (okay, constant pestering), and his ability to drive me crazy in the most entertaining way, this project would probably never have been completed. Maximilian, you're a hero – at least in your own mind.
```

Comment:
I would have expected at least one resetti reference

Reply:
fixed in [254ce8b](https://github.com/lhns/newleafradio/pull/5/commits/254ce8b69df9b93dcc96b956cac3afed420237de)